chore(styles): update tokens for form hint (#4086)
swisspost/design-system: .changeset/warm-drinks-prove.md

> ---
> '@swisspost/design-system-styles': minor
> '@swisspost/design-system-documentation': minor
> ---
> 
> Created a hint documentation page and used `post-assist` token for global style of form hints.

Reviewer: Since the component styles haven't changed externally, there's no need for a changelog in the styles package:
```suggestion
---
'@swisspost/design-system-documentation': minor
---

Created a documentation page for the form hints.
```
